### PR TITLE
sidenav will now overlay and not reflow the page

### PR DESCRIPTION
**Pull request by dumboluzz**

### CODE REVIEW

File: app/views/shared/_sidenav.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div>
+<div class="position-absolute top-0 left-0">
```
Comment (JonNorman) [inline app/views/shared/_sidenav.html.erb:1]: Do you have a picture/GIF of a before and after? Would be cool to compare them and see what the difference is.

Comment (finnstue) [reply app/views/shared/_sidenav.html.erb:1]: This is before:
before

These are after:
after1
after2

**Comment by finnstue** [comment]
@dumboluzz Can you set the right margin to be equal to the left margin?

**Comment by dumboluzz** [comment]
@finnstue Mesa fixed dha index page container besa centered. Please review!

**Comment by finnstue** [comment]
Sorry for the long pause on this, it looks great!

**Review by finnstue — APPROVED** [review]

**Pull request merged**